docs/src/main/asciidoc/reactive-routes.adoc

```diff
@@ -399,18 +399,16 @@ The previous snippet produces:
 
 You can return a `Multi` to produce a JSON Array, where every item is an item from this array.
 The response is written item by item to the client.
-The `content-type` is set to `application/json` if not set already.
-
-To use this feature, you need to wrap the returned `Multi` using `io.quarkus.vertx.web.ReactiveRoutes.asJsonArray`:
+To do that set the `produces` attribute to `"application/json"` (or `ReactiveRoutes.APPLICATION_JSON_CONTENT_TYPE`).
 
 [source, java]
 ----
-@Route(path = "/people")
+@Route(path = "/people", produces = ReactiveRoutes.APPLICATION_JSON_CONTENT_TYPE)
```

Comment:
I think that we should also mention `ReactiveRoutes.APPLICATION_JSON_CONTENT_TYPE` and friends in the `Route#produces` javadoc.

Reply:
Added, with explanations.